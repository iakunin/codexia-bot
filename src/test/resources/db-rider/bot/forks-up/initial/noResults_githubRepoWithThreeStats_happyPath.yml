github_repo:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: "150713223"
        full_name: "test-project1/test-repo1"

codexia_project:
    -
        id: 6
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 44
        coordinates: "test-project1/test-repo1"
        author: "some-first-author"
        deleted: null
        project_created_at: "2019-12-20 12:01:02"

github_repo_source:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "CODEXIA"
        external_id: "44"

github_repo_stat:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        type: "GITHUB_API"
        stat: '{"type": "GITHUB_API", "forks": 402}'
        created_at: "2020-02-09 11:07:16.654321"
    -
        id: 2
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        type: "GITHUB_API"
        stat: '{"type": "GITHUB_API", "forks": 459}'
        created_at: "2020-02-12 21:08:55.112233"
    -
        id: 4
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        type: "GITHUB_API"
        stat: '{"type": "GITHUB_API", "forks": 473}'
        created_at: "2020-02-16 18:46:09.652645"

forks_up_result: []

codexia_review: []
