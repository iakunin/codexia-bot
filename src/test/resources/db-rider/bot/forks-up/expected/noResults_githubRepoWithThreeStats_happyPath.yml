forks_up_result:
    -
        github_repo_id: 1
        github_repo_stat_id: 4

codexia_review:
    -
        codexia_project_id: 6
        author: "FORKS_UP"
        reason: "473"
        text: >-
            The repo gained 71 forks (from 402 to 473) in 1 week.
